src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -245,6 +245,7 @@ protected function getAllowedAttributes($classOrObject, array $context, $attribu
             $name = $attributeMetadata->getName();
 
             if (
+                !$attributeMetadata->getIgnore() &&
```

Comment:
given https://github.com/symfony/symfony/pull/28744/files#diff-7a86113d1484512f310d58e30eadf67dR56 is there any reason to not use `$attributeMetadata->ignore` here? That would actually implement the advertised perf benefit no?

Reply:
If we change this, I propose to change this consistently for all attributes in another PR.